json-schema-form.schema.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Thanks a lot, @jguddas for your contribution! You were the first one 😄 Approved, merging it now :)

Reply:
I think you are right, meta schema is the right term.